security-privacy-questionnaire.md

```diff
@@ -28,26 +22,22 @@ This feature does not collect or expose any such information.
 
 ## 4. How does this specification deal with sensitive information?
 
-The only sensitive information this feature handles is an opaque string that can
-be mapped to a payment instrument by an issuing bank. This instrument ID is
-saved in the user agent along side the public key credential when the issuing
-bank creates a `PaymentCredential`. Later, an origin that has access to this
-instrument ID, presumably via a trusted server integration with the issuing
-bank, can provide it to the user agent via [Payment Request API] to exercise the
-corresponding `PaymentCredential`.
+The only sensitive information this feature handles is the public ID of the
+created credential. This credential ID is saved in the user agent along side
```

Comment:
Suggest:

"The only sensitive information this feature handles relates to the (WebAuthn) credential ID of the created credential."